sdk-config.json

```diff
@@ -38,9 +38,16 @@
       "sourceOfComponents": "Server",
       "devBuild": false,
       "clientId": "10095186356008396159",
+      "clientSecret_comment": "Needed only when using 'passwordCreds' or 'clientCreds' grantType",
       "clientSecret": "",
+      "grantType_comment": "Preferred is 'authCode' as it works for public registrations",
       "grantType": "authCode",
       "redirectUri": "http://localhost:4010/",
+      "cert_and_key_comment": "path to ssl certificate file and ssl key file to use when starting local https server",
```

Comment:
For consistency, starting sentence like comments with capital letter and I think SSL should be upper case. So "Path to SSL certificate file and SSL key file..."

Reply:
HTTPS should probably be upper case, too.